config/deploy.rb

```diff
@@ -1,11 +1,11 @@
 # config valid only for Capistrano 3.1
-lock '3.2.1'
+#lock '3.2.1'
```

Comment:
I would prefer to delete these comments -- this locking business is no longer needed and was historical from the 2.x -> 3.x Capistrano upgrade as I recall.